tests/utilities.js

```diff
@@ -63,8 +63,15 @@ module.exports = {
         const expiryDateFrame = await page.frameLocator('iframe[title*="expiry date"]');
         await expiryDateFrame.getByPlaceholder('MM/YY').fill(expiryDate);
        
-        const cvcFrame = await page.frameLocator('iframe[title="Iframe for security code"]');
-        await cvcFrame.getByPlaceholder('3 digits').fill(cvc);
+        // Find iframe and fill "CVC" field
+        const cvcFrame = await page.getByRole('region[name="Credit or debit card"i]'); 
```

Comment:
Maybe the variable `cvcFrame` should be renamed to `creditDebitCardRegion`

Reply:
Renamed